Jenkinsfile

```diff
@@ -43,9 +43,7 @@ pipeline {
                 sh "mkdir -p $GOPATH/src/github.com/intel"
                 sh "cp -rf ${env.WORKSPACE} $REPO_DIR"
                 dir(path: "$GOPATH") {
-                  sh "go get -v golang.org/x/lint/golint"
-                  sh "go get -v golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow"
-                  sh "go get -v github.com/fzipp/gocyclo"
+                  sh "go get -v github.com/golangci/golangci-lint@v1.27.0"
```

Comment:
this didn't work.
Also, golangci-lint explicitly asking not to install it this way.

so, please fix following error:
```
make: golangci-lint: Command not found

Makefile:54: recipe for target 'lint' failed
```